api/test/integration/permission-tests/permission-as-limited-juris-admin-correct-juris.e2e-spec.ts

```diff
@@ -0,0 +1,1265 @@
+      });
+
+      await request(app.getHttpServer())
+        .delete(`/amiCharts`)
+        .set({ passkey: process.env.API_PASS_KEY || '' })
+        .send({
+          id: amiChartA.id,
+        } as IdDTO)
+        .set('Cookie', cookies)
+        .expect(200);
+    });
+  });
+
+  describe('Testing app endpoints', () => {
+    it('should succeed for heartbeat endpoint', async () => {
+      request(app.getHttpServer()).get('/').expect(200);
+    });
+  });
+
+  describe('Testing application endpoints', () => {
+    beforeAll(async () => {
+      await unitTypeFactoryAll(prisma);
+      await await prisma.translations.create({
+        data: translationFactory(),
+      });
+    });
+
+    it('should succeed for list endpoint for listing with no applications', async () => {
+      const listing1 = await listingFactory(jurisId, prisma);
+      const listing1Created = await prisma.listings.create({
+        data: listing1,
+      });
+
+      await request(app.getHttpServer())
+        .get(`/applications?listingId=${listing1Created.id}`)
+        .set({ passkey: process.env.API_PASS_KEY || '' })
+        .set('Cookie', cookies)
+        .expect(200);
+    });
+
+    it('should error as forbidden for retrieve endpoint', async () => {
+      const unitTypeA = await unitTypeFactorySingle(
+        prisma,
+        UnitTypeEnum.oneBdrm,
+      );
+
+      const listing1 = await listingFactory(jurisId, prisma);
+      const listing1Created = await prisma.listings.create({
+        data: listing1,
+      });
+
+      const applicationA = await prisma.applications.create({
+        data: await applicationFactory({
+          unitTypeId: unitTypeA.id,
+          listingId: listing1Created.id,
+        }),
+        include: {
+          applicant: true,
+        },
+      });
+
+      await request(app.getHttpServer())
+        .get(`/applications/${applicationA.id}`)
+        .set({ passkey: process.env.API_PASS_KEY || '' })
+        .set('Cookie', cookies)
+        .expect(403);
+    });
+
+    it('should error as forbidden for delete endpoint & create an activity log entry', async () => {
+      const unitTypeA = await unitTypeFactorySingle(
+        prisma,
+        UnitTypeEnum.oneBdrm,
+      );
+      const listing1 = await listingFactory(jurisId, prisma);
+      const listing1Created = await prisma.listings.create({
+        data: listing1,
+      });
+      const applicationA = await prisma.applications.create({
+        data: await applicationFactory({
+          unitTypeId: unitTypeA.id,
+          listingId: listing1Created.id,
+        }),
+        include: {
+          applicant: true,
+        },
+      });
+
+      await request(app.getHttpServer())
+        .delete(`/applications/`)
+        .set({ passkey: process.env.API_PASS_KEY || '' })
+        .send({
+          id: applicationA.id,
+        })
+        .set('Cookie', cookies)
+        .expect(403);
+    });
+
+    it('should succeed for public create endpoint', async () => {
```

Comment:
The new role should be able to create an application?

Reply:
This is a public application. Anyone can create an application on the public site